charts/coturn/templates/configmap-coturn-conf-template.yaml

```diff
@@ -82,3 +82,26 @@ data:
     zrest
     ## static authentication secrets will be added below this line when the
     ## runtime configuration is generated.
+
+    {{- if .Values.federate.enabled }}
+    ### federation setup
+    federation-listening-ip=__COTURN_EXT_IP__
+    federation-listening-port={{ required "Define the public port coturn should listen on for federation." .Values.federate.port }}
+    {{ if hasKey .Values.federate "dtls" }}
```

Comment:
Since values.yaml already specify some defaults, among them federate.dtls.enabled = true; this check is redundant. It also simplifies the second instance of the federation-no-dtls on line 105, which can then be removed.

```suggestion
```

Reply:
It turned out that relying on the defaults enabled some more cleanup. :smile_cat: